### PR TITLE
feat: use non linear delay for movement

### PR DESCRIPTION
**Pull request by OlliePugh**
Make the delay for picking up items not be linear.

### CODE REVIEW

File: packages/trashy_road/lib/src/game/entities/player/behaviors/player_moving_behavior.dart [hunk 1]
```diff
@@ -81,6 +84,28 @@ final class PlayerMovingBehavior extends Behavior<Player>
...
   }
 
+  /// Calculates the delay between player moves based on the inventory's size.
+  ///
+  /// This equates to roughly the following values.
+  /// 1 trash: 153ms
+  /// 2 trash: 161ms
+  /// 3 trash: 179ms
+  /// 4 trash: 217ms
+  /// 5 trash: 275ms
```
Comment (alestiago) [inline packages/trashy_road/lib/src/game/entities/player/behaviors/player_moving_behavior.dart:94]:
```suggestion
  /// 1 trash: 153ms (+0ms)
  /// 2 trash: 161ms (+8ms)
  /// 3 trash: 179ms (+26ms)
  /// 4 trash: 217ms (+64ms)
  /// 5 trash: 275ms (+122ms)
```

**Review by alestiago — APPROVED** [review]

**Comment by alestiago** [comment]
I tried it and it feels quite drastic, I think linear might work well.

**Comment by OlliePugh** [comment]
I'm not too opinionated on this, the max slow down is actually the same as before. But I think it feels a bit more intentional than linear

**Comment by alestiago** [comment]
It 100% feels more intentional.

**Review by alestiago — APPROVED** [review]

**Comment by alestiago** [comment]
Resolves #19

**Pull request merged**